packages/google_maps_flutter/google_maps_flutter/lib/src/google_map.dart

```diff
@@ -229,20 +229,20 @@ class GoogleMap extends StatefulWidget {
   /// Ground overlays to be initialized for the map.
   ///
   /// Ground overlay feature support table:
-  /// | Feature                     | Android | iOS | Web |
-  /// |-----------------------------|---------|-----|-----|
-  /// | [GroundOverlay.bounds]      | Yes     | Yes | Yes |
-  /// | [GroundOverlay.position]    | Yes     | Yes | No  |
-  /// | [GroundOverlay.width]       | Yes     | No  | No  |
-  /// | [GroundOverlay.height]      | Yes     | No  | No  |
-  /// | [GroundOverlay.anchor]      | Yes     | Yes | No  |
-  /// | [GroundOverlay.zoomLevel]   | No      | Yes | No  |
-  /// | [GroundOverlay.bearing]     | Yes     | Yes | Yes |
-  /// | [GroundOverlay.transparency]| Yes     | Yes | Yes |
-  /// | [GroundOverlay.zIndex]      | Yes     | Yes | Yes |
-  /// | [GroundOverlay.visible]     | Yes     | Yes | Yes |
-  /// | [GroundOverlay.clickable]   | Yes     | Yes | Yes |
-  /// | [GroundOverlay.onTap]       | Yes     | Yes | Yes |
+  /// | Feature                     | Android                 | iOS | Web |
+  /// |-----------------------------|-------------------------|-----|-----|
+  /// | [GroundOverlay.bounds]      | Yes                     | Yes | Yes |
+  /// | [GroundOverlay.position]    | Yes                     | Yes | No  |
+  /// | [GroundOverlay.width]       | Yes (for position only) | No  | No  |
+  /// | [GroundOverlay.height]      | Yes (for position only) | No  | No  |
+  /// | [GroundOverlay.anchor]      | Yes (for position only) | Yes | No  |
+  /// | [GroundOverlay.zoomLevel]   | No                      | Yes | No  |
```

Comment:
Should we put here (for position only) also for zoomLevel and iOS?

Reply:
yes. let's merge this and you can update the iOS side, as I am not sure about other methods either